### PR TITLE
ci(postman): Added address update test cases

### PR DESCRIPTION
**Pull request by likhinbopanna**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description
Modify the existing Stripe postman collection (Hyperswitch collection) to add the address and amount update test cases 
### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context
Billing address and shipping address Update test cases was not present in the stripe collection .

## How did you test it?
Ran the stripe collection 

image (7)

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [x] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Review by Gnanasundari24 — DISMISSED** [review]

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario22- Update address and List Payment method/Payments - Create/request.json [hunk 1]
```diff
@@ -69,12 +54,7 @@
           "state": "California",
           "zip": "94122",
           "country": "US",
-          "first_name": "joseph",
-          "last_name": "Doe"
-        },
-        "phone": {
-          "number": "8056594427",
-          "country_code": "+91"
```
Comment (pixincreate) [inline postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario22- Update address and List Payment method/Payments - Create/request.json:77]: Revert these changes

File: postman/collection-dir/stripe/QuickStart/.meta.json [hunk 2]
```diff
@@ -4,8 +4,6 @@
     "API Key - Create",
     "Payment Connector - Create",
     "Payments - Create",
-    "Payment Connector - Update",
```
Comment (pixincreate) [inline postman/collection-dir/stripe/QuickStart/.meta.json:7]: Why this was deleted?

File: postman/collection-dir/stripe/QuickStart/Payment Connector - Create/request.json [hunk 3]
```diff
@@ -44,7 +44,7 @@
       "business_label": "default",
       "connector_account_details": {
         "auth_type": "HeaderKey",
-        "api_key": "{{connector_api_key}}_invalid_values"
```
Comment (pixincreate) [inline postman/collection-dir/stripe/QuickStart/Payment Connector - Create/request.json:47]: Revert

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario23- Update Amount/Payments - Confirm/request.json [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    },
+    {
+      "key": "x-feature",
+      "value": "router-custom",
+      "type": "text",
+      "disabled": true
+    },
+    {
+      "key": "publishable_key",
+      "value": "pk_snd_8798c6a9114646f8b970b93ad5765ddf",
```
[the remark this answers is not in the document]
Comment (likhinbopanna) [reply postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario23- Update Amount/Payments - Confirm/request.json]: done

**Review by pixincreate — COMMENTED** [review]
You've reverted all the changes that was made in this PR: https://github.com/juspay/hyperswitch/pull/2531

**Review by Gnanasundari24 — APPROVED** [review]

**Review by pixincreate — APPROVED** [review]
LGTM!

**Review by SanchithHegde — APPROVED** [review]
